index.html

```diff
@@ -0,0 +1,332 @@
+                    <a href="#" class="back-end-dev">Back End Dev</a>                            
+                    <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+                    <a href="#" class="year"> 2015 </a>
+                </div>
+            </div>
+            <div class="work-detail">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+
+            <div class="tags">
+                <span class="tag">html</span>
+                <span class="tag">css</span>
+                <span class="tag">javascript</span>
+            </div>
+            <div class="action">
+                <button type="button" class="btn"><span>See Project</span> </button>
+            </div>
+        </div>
+        </div>
+
+
+        <!---------Ecnd of ard-works for Tonic-2-->
+
+
+        <!---------Card-works for multi-post stories-2-------->
+        <div class="card-works multipost-stories-2">
+            <img src="./images/Snapshoot-Portfolio-4.png" alt="Portfolio Snapshot-4" class="card-work">
+
+        <div class="left-block">
+            <div class="project">
+                <div class="project-title">                         
+                    <a href="#" class="title">Multi-purpose stories</a>
+                </div>
+                <div class="frame-2">
+                    <a href="#" class="canopy btn-link"> Canopy</a>
+                    <a href="#" class="counter"><img src="./images/Counter.png" alt="Counter image"></a>                
+                    <a href="#" class="back-end-dev btn-link">Back End Dev</a>                            
+                    <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+                    <a href="#" class="year btn-link"> 2015 </a>
+                </div>
+            </div>
+            <div class="work-detail">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+
+            <div class="tags">
+                <span class="tag">html</span>
+                <span class="tag">css</span>
+                <span class="tag">javascript</span>
+            </div>
+            <div class="action">
+                <button type="button" class="btn"><span>See Project</span> </button>
+            </div>
+        </div>
+        </div>
+        <!---------End of caard-works for multi-post stories-2------------->
+
+        <!----end of container div------------->
+        </section>
+        <!-----section-work ends here------->
+
+        <!---------About myself section------->
+        <section id="about" class="section-about-myself">
+            <div class="card-works about-myself">
+                <div class="welcome">
+                    <h1 class="name-welcome">
+                        About <br> Myself
+                    </h1>                
```

Comment:
- [ ] Kindly change this tag to a `<hx>` where x is from 2 to 6. This is because you should only have one `<h1>` element on your entire page.